dev/teamcity/dist-npm-tc

```diff
@@ -6,5 +6,3 @@ set -o errexit
 
 echo "Install dependencies"
 make install
-
-./grunt-tc clean
```

Comment:
all individual tasks clean themselves before hand now. the last remaining clean task this ran was githooks, which are tidied and maybe installed, although explicitly not on TC now, by the githooks task